enacts/flex_fcst/layout.py

```diff
@@ -9,7 +9,7 @@
 
 from . import cpt
 
-GLOBAL_CONFIG = pingrid.load_config(os.environ["CONFIG"])
+GLOBAL_CONFIG = pingrid.load_config("config-defaults.yaml:" + os.environ["CONFIG"])
```

Comment:
This is essentially what I had in mind, but it's a bit verbose and error-prone to have to copy-paste this line in so many places. I would rather have a `get_config` function defined in one place and called everywhere it's needed. Or for that matter, we might as well just have `GLOBAL_CONFIG` be defined once and then imported everywhere. `globals_.py` would be a good place for that.

Reply:
The way the application finds `config-defaults.yaml` might have to be modified to work in the docker container, but I'll cross that bridge when I come to it (i.e. when we package a new version for deployment).

Reply:
What I pushed is probably not exactly what you had in mind. The app would then be run by:

CONFIG=config-zmd.yaml:myconfig.yaml python app.py 

Maybe 3 CONFIG in the files?
CONFIG_GLOBAL
CONFIG_COUNTRY
CONFIG_APP
?

Reply:
this?

Reply:
:+1: